mpmath/function_docs.py

```diff
@@ -2237,6 +2237,12 @@
 returns a floating-point approximation. To obtain an exact
 fraction, use :func:`~mpmath.bernfrac` instead.
 
+.. versionchanged :: 1.4
+    ``bernoulli(1)`` gives ``+0.5`` instead of ``-0.5``.
+    This choice of value confers several theoretical advantages [1],
+    including an extension to complex parameters. The previous
+    behavior can be obtained with ``(-1)**n*bernoulli(n)``.
```

Comment:
```suggestion
`B_1=0.5`, this choice of value confers several theoretical advantages [1].

.. versionchanged :: 1.4
    ``bernoulli(1)`` was changed from ``-0.5`` to ``+0.5``.  The previous
    behavior can be obtained with ``(-1)**n*bernoulli(n)``.
```
I meant something like this.  Take look on the CPython sphinx docs, for example, like https://docs.python.org/3/library/functions.html#float.  The versionchanged directive is just a log entry (and it could be removed).

Reply:
Done.